src/http/actions/actions.customers.ts

```diff
@@ -73,10 +73,10 @@ export module CustomerActions {
    * @param callback {Function}
    */
   export const deleteCustomer = (customerID: string, callback: Function) => {
-    const queryString = `DELETE FROM ${_tableName} WHERE id=?`;
-    db.query(queryString, [customerID], err => {
-      if (err) {
-        callback(err);
+    const queryString = `DELETE FROM ${_tableName} WHERE id=${customerID}`;
+    _runner.run(queryString, (res)=>{
+      if (res.err) {
```

Comment:
We are using ```if``` structure calling ```callback```, and calling ```callback``` again without ```else```. I'm afraid it can produce problems in the future if ```callback``` returns control to the current code and is called twice.

Reply:
Callback is a function that returns `Void` then the code that if outside of the if will never execute if it enter inside the `if` block. Don't worry about that, it's not a problem. For now it's the best approach to our solution, if in the future we need to return a no void function, we will change it later, but actually the runner has no meaning to have an else clause inside. If you are not sure to understand what does it mean, please contact directly with me.